lib/get_pomo/mo_file.rb

```diff
@@ -24,6 +25,11 @@ def add_translations_from_text(text)
       text = StringIO.new(text)
       @translations += GetPomo::GetText::MOFile.open(text, "UTF-8").map do |msgid,msgstr|
         translation = Translation.new
+
+        if has_context? msgid
+          translation.msgctxt, msgid = msgid.split CONTEXT_SEPARATOR
+        end
```

Comment:
using the same logic in 2 places always smells like future trouble to me ... prefer
```
translation.msgctxt, msgid = msgid.split CONTEXT_SEPARATOR, 2 if msgid.include?(CONTEXT_SEPARATOR)
```